includes/admin/stripe-settings.php

```diff
@@ -33,6 +60,11 @@
 			/* translators: webhook URL */
 			'description' => $this->display_admin_settings_webhook_description(),
 		),
+		'api_credentials'               => array(
+			'title'       => __( 'API Credentials', 'woocommerce-gateway-stripe' ),
```

Comment:
Could we change API Credentials to "Stripe account keys" ?

Reply:
https://github.com/woocommerce/woocommerce-gateway-stripe/pull/1267/commits/f9ad9f38bba43311f5d8998705e2b71325202034